feature-libs/asm/asm-customer-360/sections/asm-customer-product-reviews/asm-customer-product-reviews.component.ts

```diff
@@ -59,24 +59,29 @@ export class AsmCustomerProductReviewsComponent implements OnDestroy, OnInit {
   protected subscription = new Subscription();
 
   constructor(
-    protected context: Customer360SectionContext<AsmCustomer360ReviewList>
+    protected context: Customer360SectionContext<AsmCustomer360ReviewList>,
+    protected datePipe: CxDatePipe,
+    protected translation: TranslationService
   ) {}
 
   ngOnInit(): void {
     this.subscription.add(
-      this.context.data$.subscribe((data) => {
-        this.reviewEntries = data.reviews.map((entry) => ({
-          ...entry,
-          item: combineStrings(entry.productName, entry.productCode, ', SKU: '),
-          dateAndStatus: combineStrings(
-            entry.createdAt
-              ? formatEpochTime(Number(new Date(entry.createdAt)))
-              : undefined,
-            entry.reviewStatus,
-            ' / '
-          ),
-        }));
-      })
+      combineLatest([
+        this.context.data$,
+        this.translation.translate('asm.customer360.productReviews.sku'),
+      ])
+        .pipe(
+          map(([data, skuLabel]) => {
+            this.reviewEntries = data.reviews.map((entry) => ({
+              ...entry,
+              item: `${entry.productName}, ${skuLabel} ${entry.productCode}`,
```

Comment:
We probably want the colon outside the translation string.
```suggestion
              item: `${entry.productName}, ${skuLabel}: ${entry.productCode}`,
```